code/style.css

```diff
@@ -1,7 +1,503 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+
+
+* {
+  margin: 0;
+  padding: 0;
+  
+}
+html {
```

Comment:
i think in one of lectures Jennie suggests that instead of styling <html> use <body> as html tag refers to whole html including head